config/jobs/kubernetes-csi/gen-jobs.sh

```diff
@@ -284,6 +285,16 @@ pull_optional() {
     fi
 }
 
+kubernetes_branch_name() {
+    local kubernetes="$1"
+    local repo="$2"
+    if [ "$repo" == "external-resizer" ]; then
+	echo "\"release-${kubernetes}\""
+    else
+	echo "\"${kubernetes}.0\""
+    fi
+}
```

Comment:
Please add a comment why is this necessary and it's only a temporary workaround until the next Kubernetes patch release.

Reply:
I added a comment that, this workaround will be removed once all sidecars are upgraded to use more recent tags of k8s branches.